flopy/mf6/modflow/mfsimulation.py

```diff
@@ -119,6 +131,7 @@ def load(
         verify_data=False,
         write_headers=True,
         lazy_io=False,
+        use_pandas=True,
```

Comment:
What does write_headers write?

Reply:
write_headers: bool
        When true flopy writes a header to each package file indicating that
        it was created by flopy.